### DISCUSSION TITLE
Does Zed support character variants?

### DISCUSSION THREAD

**Discussion by arjpar** [zed-industries/zed#11408] [Feature Requests]
I can't seem to find a way to enable character variants (cv01, cv02) through "buffer_font_features" in `settings.json`.

```json
"buffer_font_features": {
    "cv06": true
}
```

Doesn't work for me using Commit Mono (font).

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
